### ISSUE TITLE
If a method is obsolete then add [Obsolete] to the generated result

### ISSUE THREAD

**Issue by ghuntley** [reactiveui/ReactiveUI#1508]
**Do you want to request a *feature* or report a *bug*?**

feature

**What is the current behavior?**

EventBuilder scans assemblies in a platform and generates out `observable.fromevent` for each eventhandler for that platform.

https://github.com/reactiveui/ReactiveUI/tree/develop/src/EventBuilder

![image](https://user-images.githubusercontent.com/127353/31337360-aeebc586-ad46-11e7-9695-01fbcbc08588.png)

Sometimes these events are obsolete (or soon will be). 

**What is the motivation / use case for changing the behavior?**

Attach `[Obsolete]` on the generated events so that consumers of ReactiveUI that use `Obsolete` events inherit the designs made by the downstream framework designers.

**Comment by GKotfis** [comment]
I'll try deal with it.

**Comment by GKotfis** [comment]
I have some solution but wondering if Message and Error flag from OsboleteAttribute should be also propagated?
```
public class Class1
{
    [Obsolete()]
    public event EventHandler SampleEvent;

    [Obsolete("With message")]
    public event EventHandler SampleEventWithObsoleteMessage;
}
```
**Result:**
```
        [Obsolete]
        public IObservable<System.EventArgs> SampleEvent {
            get { return Observable.FromEventPattern<System.EventHandler, System.EventArgs>(x => This.SampleEvent += x, x => This.SampleEvent -= x).Select(x => x.EventArgs); }
        }

        [Obsolete("With message")]
        public IObservable<System.EventArgs> SampleEventWithObsoleteMessage {
            get { return Observable.FromEventPattern<System.EventHandler, System.EventArgs>(x => This.SampleEventWithObsoleteMessage += x, x => This.SampleEventWithObsoleteMessage -= x).Select(x => x.EventArgs); }
        }
```

**Comment by ghuntley** [comment]
Great idea! Any flags or messages should be passed back up.

**Comment by ghuntley** [comment]
Fixed in #1514
